irods/test/login_auth_test.py

```diff
@@ -244,12 +243,14 @@ def _setup_rodsuser_and_optional_pw(self, name, make_irods_pw = False):
             self.admin.users.remove( name )
 
     def tst0(self, ssl_opt, auth_opt, env_opt, name = TEST_RODS_USER, make_irods_pw = False):
-
+        _auth_opt = auth_opt
+        if auth_opt.startswith('pam'):
```

Comment:
Let's be explicit about which values we're checking for to better future-proof the software.

If a third PAM value appears later on for iRODS, could this result in incorrect behavior?

Reply:
I think this is to accommodate the fact that `pam` and `pam_password` are interchangeable right now in PRC for 4.3+ servers. Perhaps a comment to that effect can help future readers.

Also, tell me if I'm wrong here :p

Reply:
Right. I was thinking about how the sub-string check doesn't guard against bogus values like `pamattack`. If a bogus value makes it here, it could be possible to break something or put the PRC in a weird state.

Reply:
This is corrected now.